feat(build): use grafana officials binaries (#2)
CleverCloud/grafana-example: README.md

> # How to set up Grafana on Clever-Cloud

Reviewer: It was well written :) It's `Clever Cloud`

> 1. Use a fork of this repository as a Node.js application. It will download and run Grafana using the root `package.json` file. The install script `build.sh` will download grafana and its plugins and the start one `run.sh` will start the grafana-server binary.

Reviewer: I think the first item should be `fork this repository`. Then second should be create the application, with a link to https://www.clever-cloud.com/doc/getting-started/by-language/node/#create-an-application-on-clever-cloud

2. In your application environment variables add ```GF_SERVER_HTTP_PORT=8080```.

> 3. Add ```GRAFANA_VERSION``` and ```GRAFANA_SHA_256``` environment variables. The grafana dowload page is [here](https://grafana.com/grafana/download).

Reviewer: ```suggestion
3. Add ```GRAFANA_VERSION``` and ```GRAFANA_SHA_256``` environment variables. You can find these values on the [Grafana download page](https://grafana.com/grafana/download).
```

4. Check Node.js version used in `package.json`, for instance Grafana 7.5.5 expects Node.js 14 (see Grafana's [package.json](https://github.com/grafana/grafana/blob/v7.5.5/package.json#L317)).
5. To install custom Grafana plugin, you can set them directly in `GRAFANA_PLUGINS` environnement variable split by a `,`, for example with the value: `plugin,other_one:0.0.2,https://github.com/ovh/ovh-warp10-datasource.git`. It will install the latest version of `plugin`, the version `0.0.2` of `second_one` using the grafana-cli commands. And for the `ovh-warp10-datasource` plugin, `git clone` is used instead, as it's done for any plugin starting by `http`.

> 6. Add `GF_PLUGIN_DIR` environnement variable with `/home/bas/APP_ID/grafana/data/plugins` to use local cloned plugins.

Reviewer: Didn't you manage to make this work with a relative path? If we could avoid having the app id written there, it could be great.

> 7. You can now build and start the clever-cloud application, be aware however that if you restart it your data will not be persisted (see next section to learn how to persist data).

Reviewer: `Clever Cloud` (or you could get rid of it I guess)

> 8. To change grafana configuration through environment variables please check [grafana configuration page](https://grafana.com/docs/grafana/latest/administration/configuration/).

Reviewer: ```suggestion
8. To configure Grafana through environment variables please check [grafana configuration page](https://grafana.com/docs/grafana/latest/administration/configuration/).
```

# Persisting Data on a database

For this example we will use mysql however the procedure shouldn't be too different with any other type of database as PostgreSQL.

> 1. Create your database add-on on clever-cloud and link it to the application, you can check our addon [documentation](https://www.clever-cloud.com/doc/getting-started/quickstart/#create-your-first-add-on).

Reviewer: ```suggestion
1. Create your database add-on on Clever Cloud and link it to the application, you can check our [documentation](https://www.clever-cloud.com/doc/getting-started/quickstart/#create-your-first-add-on).
```

> 2. In your application environment variables you should now see the information you need to connect to your db.

Reviewer: ```suggestion
2. In your application environment variables you should now see the information you need to connect to your database.
```

> 3. Link them to grafana, to do this you need to create a few environment variables (replace what's in '<>' with the actual value):

Reviewer: ```suggestion
3. Link your add-on to your Grafana application. To do this you need to create a few environment variables (replace what's in '<>' with the actual value):
```

3. In your application environment variables add ```GF_SERVER_HTTP_PORT=8080```.
4. Add ```GRAFANA_VERSION``` and ```GRAFANA_SHA_256``` environment variables. You can find these values on the [Grafana download page](https://grafana.com/grafana/download).
5. Check Node.js version used in `package.json`, for instance Grafana 7.5.5 expects Node.js 14 (see Grafana's [package.json](https://github.com/grafana/grafana/blob/v7.5.5/package.json#L317)).
6. To install custom Grafana plugin, you can set them directly in `GRAFANA_PLUGINS` environnement variable split by a `,`, for example with the value: `plugin,other_one:0.0.2,https://github.com/ovh/ovh-warp10-datasource.git`. It will install the latest version of `plugin`, the version `0.0.2` of `second_one` using the grafana-cli commands. And for the `ovh-warp10-datasource` plugin, `git clone` is used instead, as it's done for any plugin starting by `http`.
7. Add `GF_PLUGIN_DIR` environnement variable with `./data/plugins` to use local cloned plugins.
8. You can now build and start the Clever Cloud application, be aware however that if you restart it your data will not be persisted (see next section to learn how to persist data).

> 8. To configure Grafana through environment variables please check [grafana configuration page](https://grafana.com/docs/grafana/latest/administration/configuration/).

Reviewer: There is twice `8` :grin:
